lldb/test/API/macosx/arm64e-attach/TestArm64eAttach.py

```diff
@@ -26,3 +31,8 @@ def test(self):
                          "target triple is updated correctly")
         error = process.Kill()
         self.assertSuccess(error)
+
+        # Test debugserver behavior.
+        self.filecheck('platform shell cat "%s"' % packets, __file__)
+        # CHECK: cputype:100000c;cpusubtype:2;ptrsize:8;ostype:macosx;vendor:apple;endian:little;
```

Comment:
this is could be done, but you could also just `self.expect("process plugin packet send qProcessInfo", substrs=["cputype:100000c;cpusubtype:2;"])`.  We may add additional keys to this response in the future so I'm not sure I like matching all of the keys.

Reply:
Thanks!